notebooks/official/explainable_ai/sdk_custom_tabular_regression_batch_explain.ipynb

```diff
@@ -61,54 +64,53 @@
       "source": [
         "## Overview\n",
         "\n",
-        "\n",
-        "This tutorial demonstrates how to use the Vertex AI SDK to train and deploy a custom tabular regression model for batch prediction with explanation.\n",
+        "This tutorial demonstrates how to use the Vertex AI SDK for Python to train and deploy a custom tabular regression model for batch prediction with explanations.\n",
         "\n",
         "Learn more about [Vertex Explainable AI](https://cloud.google.com/vertex-ai/docs/explainable-ai/overview) and [Vertex AI Batch Prediction](https://cloud.google.com/vertex-ai/docs/tabular-data/classification-regression/get-batch-predictions)."
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "objective:custom,training,batch_prediction,xai"
+        "id": "82a273206a67"
       },
       "source": [
         "### Objective\n",
         "\n",
-        "In this tutorial, you learn how to use `Vertex AI Training and Explainable AI` to create a custom image classification model with explanations, and then you learn to use `Vertex AI Batch Prediction` to make a batch prediction request with explanations.  You can alternatively create custom models using `gcloud` command-line tool or online using Cloud Console.\n",
+        "In this tutorial, you learn how to use Vertex AI training and Explainable AI to create a custom image classification model with explanations. Then, you learn how to use Vertex AI batch prediction service to make a batch prediction request with explanations.  You can alternatively create custom models using `gcloud` command-line tool or online using Cloud Console.\n",
```

Comment:
Add "the" to "Vertex AI batch prediction service"

- "...to use the Vertex AI batch prediction service..."

Change order of "alternatively":

- "Alternatively, you can create..."